bip-musig2.mediawiki

```diff
@@ -522,23 +526,19 @@ Note that the aggregate public key and list of tweaks are inputs to partial sign
 
 === Dealing with Infinity in Nonce Aggregation ===
 
-If it happens that ''is_infinite(R'<sub>i</sub>)'' inside ''[[#NonceAgg infinity|NonceAgg]]''  there is at least one dishonest signer (except with negligible probability).
-If we fail here, we will never be able to determine who it is.
-Therefore, we continue so that the culprit is revealed when collecting and verifying partial signatures.
+If it happens that ''is_infinite(R')'' inside ''[[#NonceAgg infinity|GetSessionValues]]'', either the nonce aggregator is dishonest or there is at least one dishonest signer (except with negligible probability).
+If the signer aborted in this case, it would be impossible to determine who is dishonest.
+Therefore, the signer continues so that the culprit is revealed when collecting and verifying partial signatures.
```

Comment:
nit:
```suggestion
If the signing aborted in this case, it would be impossible to determine who is dishonest.
Therefore, the signing continues so that the culprit is revealed when collecting and verifying partial signatures.
```
or could also say "the signers". I just thought that "the signer" is a little bit misleading. (Which signer?)

Reply:
ok